share/using-a-data-repository.md

```diff
@@ -7,76 +7,88 @@ category:
+* [National Archive of Criminal Justice Data] (http://www.icpsr.umich.edu/icpsrweb/NACJD/): A video datalibrary with over 6000 hours of recording on criminal justice and criminology, funded by the National Science Foundation (NSF) and the U.S. National Institutes of Health (NIH).
+* [Child & Early Education Research Connections](http://www.researchconnections.org/childcare/download): <<suggestion:include type of format/source>> Supported by the U.S. Department of Health and Human Services, this site features datasets, multimedia, briefs, tools on social science fields that range from child care and education. 
+
+##  Health Science & Rehabilitation Research
+
+* [Speech & Language Data Repository](http://sldr.org/): SLDR is a repository of oral language and multimodal data available through primary data, secondary data, tools and collections. <<suggestion: I think the site is actually called 'Ortolang'>>
+* [National Database for Autism Research](https://ndar.nih.gov/): A NIH-funded research repository that provides data on autism spectrum disorders (ASD).
+* [The SICAS Medical Imaging Repository](https://www.smir.ch/): The Swiss Institute for Computer Assisted Surgery provides a searchable database and storage of anatomical images.
+* [Systematic Review Data Repository](http://srdr.ahrq.gov/): An online tool for the extraction and management of data for systematic review or meta-analysis under the U.S. Department of Health & Human Services.
+* [Center for Large Data Research & Data Sharing in Rehabilitation](https://rehabsciences.utmb.edu/cldr/): An archiving and data sharing portal related to health care policy and developed by the University of Texas, Cornell University, and the University of Michigan.
 
-##  Health Science & Rehabilitation Research Repositories
+<<suggestion: when defining each repository, first say the source or format before describing what type of data it provides>>
 
-* [Speech & Language Data Repository](http://sldr.org/): SLDR (Speech and Language Data Repository) is a repository of oral language and multimodal data.
-* [National Database for Autism Research](https://ndar.nih.gov/): An NIH-funded research data repository that aims to accelerate progress in autism spectrum disorders (ASD) research through data sharing and the reporting of research results.
-* [The SICAS Medical Imaging Repository](https://www.smir.ch/): Searchable database and storage of anatomical images.
-* [Systematic Review Data Repository](http://srdr.ahrq.gov/): An online tool for the extraction and management of data for systematic review or meta-analysis. It also features an open archive of systematic reviews and their data. 
-* [Center for Large Data Research & Data Sharing in Rehabilitation](https://rehabsciences.utmb.edu/cldr/): An archiving and data sharing portal developed by the University of Texas, Cornell University, and the University of Michigan.
+<<suggestion: change the lists into charts, more aesthetically pleasing>>
 
```

Comment:
make edits 
remove listings by subject 
help outline materials for things to consider 
provide links to a few resources